examples/jsm/nodes/display/DotScreenNode.js

```diff
@@ -0,0 +1,127 @@
+import TempNode from '../core/TempNode.js';
+import { nodeObject, addNodeElement, tslFn, vec2, vec3, vec4 } from '../shadernode/ShaderNode.js';
+import { uniform } from '../core/UniformNode.js';
+import { NodeUpdateType } from '../core/constants.js';
+import { uv } from '../accessors/UVNode.js';
+import { texturePass } from './PassNode.js';
+import { sin, cos } from '../math/MathNode.js';
+import { add } from '../math/OperatorNode.js';
+import { Vector2, RenderTarget } from 'three';
+import QuadMesh from '../../objects/QuadMesh.js';
+
+const quadMesh = new QuadMesh();
+
+class DotScreenNode extends TempNode {
+
+	constructor( colorNode, center = new Vector2( 0.5, 0.5 ), angle = 1.57, scale = 1 ) {
+
+		super( 'vec4' );
+
+		this.colorNode = colorNode;
+		this.center = uniform( center );
+		this.angle = uniform( angle );
+		this.scale = uniform( scale );
+
+		this._renderTarget = new RenderTarget();
+		this._renderTarget.texture.name = 'dotScreen';
+
+		this._size = uniform( new Vector2() );
+
+		this._textureNode = texturePass( this, this._renderTarget.texture );
+
+		this.updateBeforeType = NodeUpdateType.RENDER;
+
+	}
+
+	getTextureNode() {
+
+		return this._textureNode;
+
+	}
+
+	setSize( width, height ) {
+
+		this._size.value.set( width, height );
+		this._renderTarget.setSize( width, height );
+
+	}
+
+	updateBefore( frame ) {
+
+		const { renderer } = frame;
+
+		const colorNode = this.colorNode;
+		const map = colorNode.value;
```

Comment:
Depending on the chain, `colorNode` might no be a `TextureNode`. Ideally, render target dimensions and type are derived differently (not from the input node).